frontend/src/pages/CalendarPage/CalendarPage.tsx

```diff
@@ -84,6 +84,16 @@ function CalendarPage() {
 
   const { isLoading, data } = useGetSchedules({ startDateTime, endDateTime });
 
+  useLayoutEffect(() => {
+    if (!dateRef.current) return;
+
+    setDateHeight(dateRef.current.clientHeight);
+  }, [dateRef.current]);
```

Comment:
> React Hook useLayoutEffect has an unnecessary dependency: 'dateRef.current'. Either exclude it or remove the dependency array. Mutable values like 'dateRef.current' aren't valid dependencies because mutating them doesn't re-render the component

👀

Reply:
제거했어요

[01f7640](https://github.com/woowacourse-teams/2022-dallog/pull/812/commits/01f764008d5f4cbe8f8bc394cc4c01f7161c83c4)

Reply:
좋습니다 dependency 배열을 아예 없애도 괜찮은가요?!

Reply:
```tsx
  useLayoutEffect(() => {
    if (!(dateRef.current instanceof HTMLDivElement)) return;

    setMaxScheduleCount(
      Math.floor(
        (dateRef.current.clientHeight - SCHEDULE.HEIGHT * 4) / (SCHEDULE.HEIGHT_WITH_MARGIN * 4)
      )
    );
  }, [setMaxScheduleCount]);
```

린트 해결을 위해 setState함수를 넣으라는것 이외에는 문제가 없어요.

근데 지금 데브 서버가 안되어서 체크 못하고 있어요.

Reply:
오호 그러고 보니 dateRef.current.clientHeight가 각 달마다 다르네요!
같은데 굳이 왜 넣어주지 생각했어요 🫢
dateRef.current 넣어주는 거 좋아요~!